src/zookeeper/deploy/start.sh

```diff
@@ -24,16 +24,8 @@ pushd $(dirname "$0") > /dev/null
 kubectl apply --overwrite=true -f zookeeper.yaml || exit $?
 
 
-PYTHONPATH="../../../deployment" python -m  k8sPaiLibrary.monitorTool.check_node_label_exist -k zookeeper -v "true"
-ret=$?
-
-if [ $ret -ne 0 ]; then
-    echo "No Zookeeper Pod in your cluster"
-else
-    # wait until all zookeeper are ready.
-    PYTHONPATH="../../../deployment" python -m  k8sPaiLibrary.monitorTool.check_pod_ready_status -w -k app -v zookeeper || exit $?
-fi
-
+# wait until all zookeepers are ready.
```

Comment:
Before the ready check, we'd better sleep a minutes

Reply:
ok I think a minutes is a bit long, I set it to 30s